solr/setup.rst

```diff
@@ -27,23 +31,28 @@ Create a buildout (*buildout.cfg*) which installs both requirements
     [buildout]
     extends =
         plone5.cfg
-        solr.cfg
-        solr4.cfg
+    parts +=
+        solr
 
     [instance]
     eggs +=
         collective.solr
 
+    [solr]
+    recipe = kitconcept.recipe.solr
+    src = http://archive.apache.org/dist/lucene/solr/8.2.0/solr-8.2.0.tgz
+    solr-config = config
+
     [versions]
-    collective.solr = 6.0a1
-    collective.recipe.solrinstance = 6.0.0b3
+    collective.solr = 8.0.0a5
+    kitconcept.recipe.solr = 1.0.0a5
 
 
 Run buildout:
 
 .. code-block:: console
 
-   python2.7 bootstrap-buildout.py
+   python bootstrap-buildout.py
```

Comment:
Is this still state of the art, or virtualenv / pip?

Reply:
Ah, you're right, that should probably be updated as well in the next round. Thanks!